content/master/concepts/managed-resources.md

```diff
@@ -5,20 +5,19 @@ weight: 102
 
 A Managed Resource (MR) is Crossplane's representation of a resource in an
 external system - most commonly a cloud provider. Managed Resources are
-opinionated, Crossplane Resource Model ([XRM][term-xrm]) compliant Kubernetes
-Custom Resources that are installed by a Crossplane [provider].
+opinionated, Crossplane Resource Model ([XRM]({{<ref "../concepts/terminology">}})) compliant Kubernetes
+Custom Resources that are installed by a Crossplane [provider]({{<ref "providers" >}}).
 
 For example, `RDSInstance` in the AWS Provider corresponds to an actual RDS
 Instance in AWS. There is a one-to-one relationship and the changes on managed
 resources are reflected directly on the corresponding resource in the provider.
 Similarly, the `Database` types in the SQL provider represent a PostgreSQL or
-MySQL database. You can browse [API Reference][api-reference] to discover all
-available managed resources.
+MySQL database.
```

Comment:
This drops the broken link. This chapter is due for a rewrite in the near future and we'll properly link to a CRD API reference.